woocommerce/wc-admin
# Style and icon updates for Activity Panel

Issues addressed:

* Icon alignment
* Icon updates to match latest designs
* Removed some overrides (https://github.com/woocommerce/woocommerce-admin/issues/3897)

Before: 

![before](https://cldup.com/iajggpHJw5-3000x3000.png)

After:

![after](https://cldup.com/atAmy3_m_Q-3000x3000.png)

# To test
View any screen with the Activity Panel tabs present. Ensure the tabs work as expected.
